proto/prysm/v1alpha1/block/mock.go

```diff
@@ -0,0 +1,190 @@
+package block
+
+import (
+	ssz "github.com/ferranbt/fastssz"
+	types "github.com/prysmaticlabs/eth2-types"
+	enginev1 "github.com/prysmaticlabs/prysm/proto/engine/v1"
+	ethpb "github.com/prysmaticlabs/prysm/proto/prysm/v1alpha1"
+	"google.golang.org/protobuf/proto"
+)
+
+type MockSignedBeaconBlock struct {
```

Comment:
The file is only used by tests, right? If so, we should name it `mock_test` to exclude it from production code.

Reply:
The problem is, it is intended to be used by other packages.

Reply:
In that case it should live in its own package. See https://github.com/prysmaticlabs/prysm/tree/develop/cmd/mock for an example.

Reply:
fair enough! I also like that idea because it would be helpful to consolidate code that generates block fixtures. `proto/prysm/v1alpha1/block/mock` would be an intuitive place to find that kind of thing.